mne/forward/_make_forward.py

```diff
@@ -470,8 +472,10 @@ def _prepare_for_forward(src, mri_head_t, info, bem, mindist, n_jobs,
         raise RuntimeError('No MEG or EEG channels found.')
 
     # pick out final info
+    comps, info['comps'] = info['comps'], []  # avoid picking errors
     info = pick_info(info, pick_types(info, meg=meg, eeg=eeg, ref_meg=False,
                                       exclude=[]))
+    info['comps'] = comps
```

Comment:
It depends on when `_check_consistency` gets called, and it must not for `forward`, otherwise tests would not pass (assuming we do an I/O test for CTF, which I hope we do!).

If we add a consistency check we can make an exception like `._check_consistency(check_comps=False)`